css/bootstrap/_jumbotron.scss

```diff
@@ -0,0 +1,17 @@
+.jumbotron {
```

Comment:
In "good old times", I would have recommended "bower install" to avoid checkin-in third party thigns into a repo. I dunno how this is done today. `yarn`?

Reply:
Came bundled with the bootstrap package from www.jabref.org. Dunno...